meta/downloadHelperBinaries.ts

```diff
@@ -139,7 +139,18 @@ async function downloadNile() {
 }
 
 async function downloadComet() {
-  return downloadGithubAssets(
+  await downloadGithubAssets(
+    'GalaxyCommunication',
+    'imLinguin/comet',
+    RELEASE_TAGS['comet'],
+    {
+      x64: {
+        win32: 'GalaxyCommunication-dummy.exe'
+      },
+      arm64: {}
+    }
+  )
+  await downloadGithubAssets(
```

Comment:
The two promises being awaited here should ideally run in parallel to fully utilize the network bandwidth

```ts
return Promise.all([
  downloadGithubAssets('GalaxyCommunication', /* ... */ ),
  downloadGithubAssets('comet', /* ... */ )
])